src/main/java/com/webonise/rbs/RbsApplication.java

```diff
@@ -1,12 +1,16 @@
 package com.webonise.rbs;
 
+import com.webonise.rbs.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
```

Comment:
why have we imported these new classes to RbsApplication. I dont see their usage anywhere.